solc/CommandLineInterface.cpp

```diff
@@ -424,13 +434,9 @@ void CommandLineInterface::handleGasEstimation(string const& _contract)
 
 void CommandLineInterface::readInputFiles()
 {
-	solAssert(!m_standardJsonInput.has_value(), "");
```

Comment:
Just curious because I've seen it all over the place - is this equivalent to your altered version where you don't pass `""`? I.e. will this just terminate with an empty message?

Reply:
Yep, this is equivalent to an empty message.

Reply:
Until quite recently you could not skip the message (#12019). And it's just a tiny detail, so I don't think it makes sense to spend time replacing it all everywhere. We just sometimes remove the empty messages when we clean up code.